Sourcery/Generating/Template/JavaScript/JavaScriptTemplate.swift

```diff
@@ -20,11 +20,34 @@ final class JavaScriptTemplate: Template {
         let path = Bundle(for: JavaScriptTemplate.self).path(forResource: "ejsbundle", ofType: "js")!
         let ejs = try String(contentsOfFile: path, encoding: .utf8)
 
+        var errorContent = ""
```

Comment:
Maybe it's better to use `Error` instead of string and just throw it if it's not nil, or return content?